tests/test_orm_model.py

```diff
@@ -193,6 +193,27 @@ def test_passthrough(methodname):
 
     mock_endpoint.assert_called_once_with(a=1, b=2, c=3)
 
+    
+def test_disallowed_args():
+    """
+    Test the argument sanitizer, `use_field_ids` attribute, and disallowed kwargs
+    """
```

Comment:
This looks like it exercises everything; however, since it's heavily mocked, we're never actually testing that a return payload using field IDs (instead of names) will be mapped correctly to the ORM's fields. Maybe we can find a way to do that?

Reply:
Let me know if the testing works, I had to add  a separate `fake_meta_from_ids` and a separate model `FakeModelByIds`. If there is more professional way to do this let me know?